This is strange. Travis doesn't generate the test case error that I made to proof the error. But it fails locally as it should:

```
    @mock.patch('kolibri.utils.version.get_git_describe', return_value="v0.1.0-alpha1-123-abcdfe12")
    def test_alpha_0_consistent_git(self, describe_mock):
        """
            Test that we get the git describe data when it's there
            """
>       assert get_version((0, 1, 0, "alpha", 0)) == "0.1.0a1.dev+git-123-abcdfe12"
E       AssertionError: assert '0.1.0.dev1.d...-123-abcdfe12' == '0.1.0a1.dev+git-123-abcdfe12'
E         - 0.1.0.dev1.dev+git-123-abcdfe12
E         ?      ^^^^
E         + 0.1.0a1.dev+git-123-abcdfe12
E         ?      ^

kolibri/utils/tests/test_version.py:153: AssertionError
```

Overwriting with some changes and using this PR for a bit of debugging...

--- a/kolibri/utils/version.py
+++ b/kolibri/utils/version.py
@@ -265,8 +265,6 @@ def get_prerelease_version(version):
     """
 
     mapping = {'alpha': 'a', 'beta': 'b', 'rc': 'rc'}
-    if version[4] == 0 and version[3] == 'alpha':
-        mapping['alpha'] = '.dev'
 
     major = get_major_version(version)
     major_and_release = major + mapping[version[3]] + str(version[4])
@@ -366,6 +364,12 @@ def get_prerelease_version(version):
                 )
             )
         return version_file
+
+    # Finally, if there was no git data or VERSION file, map the alpha-0 to
+    # .dev
+    if version[4] == 0 and version[3] == 'alpha':
+        mapping['alpha'] = '.dev'
+        major_and_release = major + mapping[version[3]] + str(version[4])
 
     # In all circumstances, return the initial findings
     return major_and_release

